README.md

```diff
@@ -86,12 +169,14 @@ final class Foo
  * You can find more information about us on https://bitbag.shop and write us
  * an email on mikolaj.krol@bitbag.pl.
  */
+
+ declare(strict_types=1);
  
  namespace Foo;
  
  use Foo\Bar\App;
  
- final class Bar implements BarInterface
+ final class BazBar implements Bar
```

Comment:
Can't we stay with the `BarInterface` convention? It's so intuitive once you got used to Sylius 🙏